hedera-node/src/main/java/com/hedera/services/state/virtual/schedule/ScheduleSecondVirtualValue.java

```diff
@@ -106,6 +123,11 @@ public void deserialize(SerializableDataInputStream in, int version) throws IOEx
 				ids.put(RichInstant.from(in), l.toImmutable());
 			}
 		}
+		if (in.readByte() == 1) {
```

Comment:
`MerkleMap` will fail hard if a value is ever deserialized with a `null` key, so I don't think it makes sense to have a workaround for this case.

Reply:
It would be null in the VritualMap case

Reply:
When going back to `VirtualMap` this will no longer be `Keyed` though, so wouldn't need to workaround anything there either. (Presumably we will update the `Versioned.getVersion()` when going back to `VirtualMap`.)

Reply:
But at no point will it be correct to serialize or deserialize a `null` key.